test/cmp_rt2bl.py

```diff
@@ -43,19 +204,39 @@ def main():
                 error_count = error_count + 1
             else:
                 print("Output for "+run["case"]+"_"+run["suite"]+ " is IDENTICAL to baseline")
+            # end if
+
+            # Create plots between RTs and baselines
+            plot_files = plot_results(file_rt,file_bl)
```

Comment:
Can we move this to only create plots if results are different than baselines? Otherwise, it's kinda wasted effort.

Reply:
Good point. Will do

Reply:
done.